src/recommendationservice/recommendation_server.py

```diff
@@ -88,6 +83,9 @@ def get_product_list(request_product_ids):
         indices = random.sample(range(num_products), num_return)
         # Fetch product ids from indices
         prod_list = [filtered_products[i] for i in indices]
+        
+        span.set_attribute("app.filtered_products.list", prod_list)
```

Comment:
```suggestion
        num_products = len(prod_list)
        span.set_attribute("app.filtered_products.count", num_products)
```
According to: https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/manual_span_attributes.md#recommendationservice
 The attribute is called `app.filtered_products.count` and it is a number.

We can just update the docs if we would like to keep the new name and have it as a list.

Reply:
Not sure what the issue is here? There's both a `span.set_attribute("app.filtered_products.list", prod_list)` and `span.set_attribute("app.filtered_products.count", num_products)`

Reply:
To clarify: I am not replacing `app.filtered_products.count` with  `app.filtered_products.list`. All I did was add another span attribute, `app.filtered_products.list`. If that is causing too much confusion, I can remove it.

Reply:
Why add this attribute, if add all products items to this attributes, will the attributes be too verbosely? And I think this pr is about adding metrics, so the change of trace may better move to another new pr?